libs/service-portal/graphql/src/lib/queries/getActorDelegations.ts

```diff
@@ -0,0 +1,10 @@
+import { gql } from '@apollo/client'
+
+export const ACTOR_DELEGATIONS = gql`
+  query ActorDelegations {
+    authActorDelegations {
```

Comment:
Query the `id` as well for the apollo store.

We should just add the
```
graphql/required-fields
```
eslint rule where it's required to query the ID of a type.
If the ID isn't queried, the apollo store doesn't know how to construct its objects based on the data already in the cache.

Reply:
Yeah, the access control pages use different auth-public-api endpoints that only manage custom delegations. The GraphQL layer should return CustomDelegations that have the non-null id field.

Reply:
Also, this queries "actor delegations" while the access control queries "subject delegations". These should not intersect.